javascript/javascript1/week1/index.js

```diff
@@ -0,0 +1,61 @@
+//Age-ify (A future age calculator)
+
+let yearOfBirth = 1991;
+let yearFuture = 2027;
+let age = (yearFuture - yearOfBirth);
+console.log("You will be " + age + " years old in 2027");
+
+
+//Goodboy-Oldboy (A dog age calculator)
+//Here I assume something should've been done differently...
+let dogYearOfBirth = 1991;
+let dogYearFuture = 2027;
+let dogYear = (dogYearFuture - dogYearOfBirth);
+let shouldShowResultInDogYears = (dogYear / 7);
+function dogAge () {
+if (shouldShowResultInDogYears == true) {
+  console.log("Your dog will be " + shouldShowResultInDogYears + " dog years old in 2027");
+} else {
+    console.log("Your dog will be " + dogYear + " human years old in 2027");
+}
+};
+
+
+
+//Housey pricey (A house price estimator)
+//For Peter
+let height = 10;
+let width = 8;
+let depth = 10;
+let volumeInMeters = height * width * depth;
+let gardenSizeInM2 = 100;
+let housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+console.log(housePrice);
+if
```

Comment:
"if" should be in the same line of the condition, like this 
if(some condition ) 
   do something 
    }
and no need to semicolon " ; " after the condition :- )